package.json

```diff
@@ -50,6 +50,7 @@
     "@digitalbazaar/multikey-context": "^2.0.1",
     "@digitalbazaar/vc": "github:digitalbazaar/vc#vc-2.0-time-props",
     "@digitalcredentials/did-context": "^1.0.0",
+    "@bedrock/did-io": "^10.0.0",
```

Comment:
one small thing here. I'm assuming this works, but the `@bedrock` libs are designed to work inside of a bedrock instance. The test suites are 100% not bedrock. You could try using the dependent library: https://github.com/digitalbazaar/bedrock-did-io/blob/add41972383ddb0c265c9ecd55e2b41bec99809b/package.json#L31

`@digitalbazaar/did-io`